iocore/net/SSLCertLookup.cc

```diff
@@ -160,7 +160,40 @@ ticket_block_alloc(unsigned count)
 
   return ptr;
 }
+ssl_ticket_key_block *
+ssl_create_ticket_key_block_buffer(char *ticket_key_data, int ticket_key_len)
+{
+  ssl_ticket_key_block *keyblock = NULL;
+  int num_ticket_keys            = ticket_key_len / sizeof(ssl_ticket_key_t);
+  if (num_ticket_keys == 0) {
+    Error("SSL session ticket key is too short (>= 48 bytes are required)");
+    goto fail;
+  }
+
+  // Increase the stats.
+  if (ssl_rsb != NULL) { // ssl_rsb is not initialized during the first run.
+    SSL_INCREMENT_DYN_STAT(ssl_total_ticket_keys_renewed_stat);
+  }
```

Comment:
Sure, but the single responsibility of this function is to load a ticket block. If the caller needs a metric to track how many times it is called, the caller should do it. By pushing higher level responsibilities into library code, we are breaking the single responsibility principle and making the code less reusable and harder to reason about.